Activities/Cryptography/UiPath.Cryptography.Activities/UiPath.Cryptography.Activities.csproj

```diff
@@ -53,6 +53,7 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <Folder Include="NetCore\" />
```

Comment:
why add this? what is the reason? should not be here

Reply:
Removed